test/index.js

```diff
@@ -1,12 +1,13 @@
 var assert = require('assert');
 var test = require('testit');
 var parser = require('../');
-var parse = parser;
+var parse = parser.parse;
+var StackElement = parser.StackElement;
 
 test('parse', function () {
   test('works out how much depth changes', function () {
     var state = parse('foo(arg1, arg2, {\n  foo: [a, b\n');
-    assert.deepEqual(state.stack, [ ')', '}', ']' ]);
+    assert.deepEqual(state.stack, [ StackElement.ROUND_BRACKET, StackElement.CURLY_BRACKET, StackElement.SQUARE_BRACKET ]);
```

Comment:
> One of the main advantages to using numbers is that it forces consumers to actually use the `TOKEN_TYPES` enum.

Is that really an advantage though? As a JS-only user yet to be enlightened by TypeScript, using the longer names is not only inconvenient, but can also be error-prone ("is it ROUND_BRACKET_RIGHT or RIGHT_ROUND_BRACKET or RIGHT_PARENTHESIS?").